rakelib/pwsh.rake

```diff
@@ -26,13 +26,25 @@ namespace :pwsh do
     }
 
     @hardcoded_cmdlets = {
-      'createkeys' => {
+      'secret:createkeys' => {
         'verb' => 'New',
         'noun' => 'BoltSecretKey'
       },
-      'show-modules' => {
+      'puppetfile:show-modules' => {
         'verb' => 'Get',
         'noun' => 'BoltPuppetfileModules'
+      },
+      'puppetfile:generate-types' => {
+        'verb' => 'Register',
+        'noun' => 'BoltPuppetfileTypes'
+      },
+      'module:generate-types' => {
+        'verb' => 'Register',
+        'noun' => 'BoltModuleTypes'
+      },
+      'module:show-modules' => {
+        'verb' => 'Get',
+        'noun' => 'BoltModule'
       }
```

Comment:
We shouldn't need to hardcode this, since the command is `bolt module show`

Reply:
I think that's from @jpogran

Reply:
I'll try without it, but coulda sworn..

Reply:
Good catch @beechtom , fixing